### DISCUSSION TITLE
[Feature]: Tapping Photos brings user to top of Photo timeline

### DISCUSSION THREAD

**Discussion by danejur** [immich-app/immich#1590] [Feature Request]
### Feature detail

When the user is scrolled down the Photos timeline, tapping the Photos button again should bring the user back to the top of the timeline.

### Platform

Mobile App

**Comment by bo0tzz** [comment]
On iOS a common action to scroll to the top is tapping at the top of the screen.

**Comment by miguelpruivo** [comment]
This needs to be addressed by either using or setting the `PrimaryScrollController` in the timeline screen.

**Comment by YarosMallorca** [comment]
This has been implemented in the mobile app (new timeline), at least on the main "Photos" page it works.
